tests/Aspire.Hosting.Redis.Tests/AddRedisTests.cs

```diff
@@ -208,7 +204,7 @@ public async Task MultipleRedisInstanceProducesCorrectRedisHostsVariable(string
             DistributedApplicationOperation.Run,
             TestServiceProvider.Instance);
 
-        Assert.Equal($"myredis1:{containerHost}:5001:0,myredis2:host2:5002:0", config["REDIS_HOSTS"]);
+        Assert.Equal($"myredis1:{redis1.Resource.Name}:6379:0,myredis2:myredis2:6379:0", config["REDIS_HOSTS"]);
```

Comment:
I don't quite understand why these ports are changing for these tests?

Reply:
Container to container requires using the in-container target port instead of the public port for connections